colour/colorimetry/spectrum.py

```diff
@@ -839,7 +835,7 @@ def items(self):
             Spectral power distribution data generator.
         """
 
-        return self.__iter__()
+        return np.array(list(self.__iter__()))
```

Comment:
Could the intermediary list be avoided by using [`np.fromiter`](https://docs.scipy.org/doc/numpy/reference/generated/numpy.fromiter.html)?